source/base/polynomials_nedelec_sz.cc

```diff
@@ -0,0 +1,188 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2015 - 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+#include <deal.II/base/polynomials_nedelec_sz.h>
+
+DEAL_II_NAMESPACE_OPEN
+namespace
+{
+  Threads::Mutex coefficients_lock;
```

Comment:
Not wrong, but it struck me while looking through the .h file that we probably need this lock but didn't see it. Can you move the declaration of this variable into the class as a static member variable?